hack/add-vclusters.sh

```diff
@@ -0,0 +1,41 @@
+environments=("development" "staging" "production")
```

Comment:
I changed this to only be "development" and "production" because I noticed 3 environments was using too many resources